### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by siddsuresh97** [kyrs/NCC-experiments#3]
Could you please update the readme so that I can reuse your code

**Comment by kyrs** [comment]
yes It's in my TODO but for some reason I am not getting enough time to finish it off. If you need help in implementation. do contact other users who have raised an issue. they might have done it. From my end it will be tough to add it by this month.

**Comment by siddsuresh97** [comment]
Ok sure!
